[Keycloak Integration CI] - Extension - Start keycloak failed

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
ERROR: Failed to start server in (development) mode
ERROR: Cannot invoke "Object.getClass()" because "val1" is null
For more details run the same command passing the '--verbose' option. Also you can use '--help' to see the details about the usage of the particular command.
Error: Process completed with exit code 4.
```
https://github.com/keycloak/keycloak-quickstarts/actions/runs/13536244746/job/37828366581#step:7:453

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java
+++ b/quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java
@@ -383,7 +383,7 @@ class KeycloakProcessor {
                 Properties properties = descriptor.getProperties();
                 // register a listener for customizing the unit configuration at runtime
                 runtimeConfigured.produce(new HibernateOrmIntegrationRuntimeConfiguredBuildItem("keycloak", descriptor.getName())
-                        .setInitListener(recorder.createUserDefinedUnitListener(properties.getProperty(AvailableSettings.JAKARTA_JTA_DATASOURCE))));
+                        .setInitListener(recorder.createUserDefinedUnitListener(properties.getProperty(AvailableSettings.DATASOURCE))));
                 userManagedEntities.addAll(descriptor.getManagedClassNames());
             }
         }

--- a/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakRecorder.java
+++ b/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakRecorder.java
@@ -179,7 +179,7 @@ public class KeycloakRecorder {
                                 return name;
                             }
                         })) {
-                    propertyCollector.accept(AvailableSettings.JAKARTA_JTA_DATASOURCE, instance.get());
+                    propertyCollector.accept(AvailableSettings.DATASOURCE, instance.get());
                 }
             }
         };

